tests/python/unittest/test_meta_schedule_tune_tir.py

```diff
@@ -32,6 +32,96 @@
 logging.basicConfig()
 logging.getLogger("tvm.meta_schedule").setLevel(logging.DEBUG)
 
+# fmt: off
+@tvm.script.ir_module
```

Comment:
please move this test to `tests/python/unittest/test_meta_schedule_space_cuda_winograd.py`